src/pudl/load/sqlite.py

```diff
@@ -0,0 +1,43 @@
+"""Load PUDL data into an SQLite database."""
+
+import logging
+
+import sqlalchemy as sa
+
+from pudl.metadata.classes import Package, Resource
+from pudl.metadata.resources import RESOURCE_METADATA
+
+logger = logging.getLogger(__name__)
+
+
+def dfs_to_sqlite(dfs, engine, foreign_keys="OFF"):
+    """Load a dictionary of dataframes into the PUDL DB."""
+    # This magic makes SQLAlchemy tell SQLite to check foreign key constraints
+    # whenever we insert data into thd database, which it doesn't do by default
+    @sa.event.listens_for(sa.engine.Engine, "connect")
+    def _set_sqlite_pragma(dbapi_connection, connection_record):
+        from sqlite3 import Connection as SQLite3Connection
+        if isinstance(dbapi_connection, SQLite3Connection):
+            cursor = dbapi_connection.cursor()
+            cursor.execute(f"PRAGMA foreign_keys={foreign_keys};")
+            cursor.close()
+
+    # Build a list describing all possible PUDL DB tables:
+    resources = [Resource.from_id(x) for x in RESOURCE_METADATA]
```

Comment:
That is how it's currently set up. It doesn't have to be that way, but it does seem like our default behavior at this point is to load everything, and we aren't supporting others in trying to create their own wacky incomplete setups. And I don't know if there's any harm in having empty tables, is there?

Reply:
yeah i think this is fine. it is 100% not what we were doing in the data package context but that added a tooonnnn of additional code and complexity. unless it feels like a requirement down the line i think this is a fine behavior

Reply:
I looked at this again and it was easy to make it just insert the tables that exist in the metadata object, so I switched to that behavior since it seems more internally consistent.